src/control/lib/netdetect/netdetect_integration_test.go

```diff
@@ -34,53 +33,73 @@ import (
 	"testing"
 	"time"
 
+	"github.com/pkg/errors"
+
 	"github.com/daos-stack/daos/src/control/common"
 	"github.com/daos-stack/daos/src/control/logging"
 )
 
 const maxConcurrent = 1000
 
-func getSocketID(t *testing.T, pid int32, wg *sync.WaitGroup) {
+func getSocketID(t *testing.T, pid int32, wg *sync.WaitGroup, ctx context.Context) error {
```

Comment:
Convention calls for the context to be passed as the first parameter to functions. This will trigger a linter warning.

Reply:
fixed